docs/products/opensearch/howto/os-metrics.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Aiven for OpenSearch® metrics available via Prometheus
+sidebar_label: OpenSearch metrics with Prometheus
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import HostMetrics from "@site/static/includes/host-metrics.md";
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import LimitedBadge from "@site/src/components/Badges/LimitedBadge";
+
+Monitor and optimize your Aiven for OpenSearch service with metrics available via Prometheus.
+These metrics help track cluster health, replication status, and overall performance.
+
+## Prerequisites
+
+- Enable Prometheus integration. For setup instructions, see
+  [Prometheus integration setup](/docs/platform/howto/integrations/prometheus-metrics).
+- Note the Prometheus **username** and **password** in the **Integration endpoints**
+  section of the  [Aiven console](https://console.aiven.io/).
+
+## Access Prometheus metrics
+
+<Tabs groupId="group1">
+<TabItem value="browser" label="Browser" default>
+
+1. Open your service's <ConsoleLabel name="overview"/> page in the
+   [Aiven console](https://console.aiven.io/).
+1. In the **Connection information** section, click the **Prometheus** tab.
+1. Copy the **Service URI**.
+1. Paste the Service URI into your browser's address bar.
+1. When prompted, enter your Prometheus credentials.
+1. Click **Login**.
+
+</TabItem>
+<TabItem value="curl" label="Curl">
```

Comment:
While cURL is indeed a CLI tool, the example provided explicitly uses the curl command. To avoid confusion and ensure precision, I’ve updated the label to **"Retrieve via cURL"**, which better reflects the command's context.